From e9ad950f7838921b1d0c85b503d7c9668be663b6 Mon Sep 17 00:00:00 2001
From: Michael Orlitzky <michael@orlitzky.com>
Date: Thu, 22 Jun 2023 16:57:55 -0400
Subject: [PATCH 1/1] check/Makefile.am: disable two failing tests.

Mentioned on Github at,

  https://github.com/sagemath/sage/pull/35745#issuecomment-1603298668

For the moment, and without more information, disabling them is the
simplest thing to do.
---
 check/Makefile.am | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/check/Makefile.am b/check/Makefile.am
index ef0aa03..be1260c 100644
--- a/check/Makefile.am
+++ b/check/Makefile.am
@@ -1,10 +1,10 @@
 TESTS = 	chk_integrate  \
 		chk_factor chk_normalize chk_partfrac chk_limit \
 		chk_cas chk_geo chk_morley_demo chk_xavier \
-		chk_fhan1 chk_fhan2 chk_fhan3 chk_fhan4 chk_fhan5 \
+		chk_fhan1 chk_fhan2 chk_fhan3 chk_fhan5 \
 		chk_fhan6 chk_fhan8 chk_fhan0 \
 		chk_fhan11 chk_fhan12 chk_fhan13 chk_fhan14 chk_fhan15 \
-		chk_fhan16 chk_fhan17 chk_fhan19 chk_fhan20 \
+		chk_fhan16 chk_fhan19 chk_fhan20 \
 		chk_fhan21 chk_fhan9 chk_fhan18 chk_fhan7
 EXTRA_DIST = 	chk_integrate chk_intinda chk_intindbo chk_intindbr \
 		chk_intindc chk_intindj chk_intindm \
-- 
2.39.3

